src/test/javascript/spec/component/course/course-management-container.component.spec.ts

```
@@ -0,0 +1,753 @@
+    id: 1,
+    title: 'Course1',
+    exams,
+    exercises: [exercise1],
+    isAtLeastInstructor: true,
+    isAtLeastEditor: true,
+    isAtLeastTutor: true,
+    description:
+        'Nihilne te nocturnum praesidium Palati, nihil urbis vigiliae. Salutantibus vitae elit libero, a pharetra augue. Quam diu etiam furor iste tuus nos eludet? ' +
+        'Fabio vel iudice vincam, sunt in culpa qui officia. Quam temere in vitiis, legem sancimus haerentia. Quisque ut dolor gravida, placerat libero vel, euismod.',
+    courseInformationSharingConfiguration: CourseInformationSharingConfiguration.COMMUNICATION_AND_MESSAGING,
+    courseIconPath: 'path/to/icon.png',
```

Biome (1.9.4)

[error] This property value named courseIconPath is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named courseIconPath


(lint/suspicious/noDuplicateObjectKeys)